djangoguestbook/static/js/guestbook/tests/models/GreetingStore.js

```diff
@@ -0,0 +1,164 @@
+define([
+	"doh/runner",
+	"dojo/json",
+	"/static/js/sinon/sinon_1_12_1.js",
+	"/static/js/guestbook/models/GreetingStore.js"
+], function(doh, json, sinon_1_12_1, GreetingStore){
+
+	doh.register("TestGreetingStore", [
+		{
+			name: "Create_Greeting_Successful",
+			setUp: function(){
+				this.GreetingStore = new GreetingStore();
```

Comment:
mỗi testcase đều sử dụng biến mới, không lưu biến lại trong this object, trừ khi có lý do đặc biệt